conf/references/mlss_conf.xml

```diff
@@ -253,55 +269,191 @@
+      <!-- Cnidaria -->
+      <genome name="acropora_millepora_gca013753865v1"/>
 
-      <!-- Hemiptera -->
-      <genome name="bemisia_tabaci_asiaii5"/>
-      <genome name="acyrthosiphon_pisum"/>
-      <genome name="rhodnius_prolixus"/>
-      <genome name="trialeurodes_vaporariorum_gca011764245"/>
-      <genome name="cimex_lectularius"/>
+      <!-- Annelida -->
+      <genome name="helobdella_robusta"/>
+      <genome name="owenia_fusiformis_gca903813345v1"/>
+      <!-- Brachiopoda -->
+      <genome name="lingula_anatina_gca001039355v2"/>
+      <!-- Mollusca -->
+      <genome name="crassostrea_virginica_gca002022765v4"/>
+      <genome name="mercenaria_mercenaria_gca014805675v2"/>
+      <genome name="patella_pellucida_gca917208275v1"/>
+      <genome name="octopus_sinensis_gca006345805v1"/>
+      <genome name="pomacea_canaliculata_gca003073045v1"/>
+      <!-- Nemertea -->
+      <genome name="lineus_longissimus_gca910592395v2"/>
+      <!-- Platyhelminthes -->
+      <genome name="hymenolepis_microstoma"/>
+      <genome name="clonorchis_sinensis_gca003604175v2wb"/>
+      <genome name="schmidtea_mediterranea_gca002600895v1wb"/>
+      <genome name="schistosoma_mansoni"/>
+      <genome name="echinococcus_multilocularis_gca000469725v3wb"/>
+      <!-- Priapulida -->
+      <genome name="priapulus_caudatus_gca000485595v2"/>
+      <!-- Tardigrada -->
+      <genome name="hypsibius_exemplaris_gca002082055v1"/>
+      <!-- Rotifera -->
+      <genome name="adineta_vaga"/>
+      <!-- Nematoda -->
+      <genome name="ascaris_suum"/>
+      <genome name="brugia_malayi"/>
+      <genome name="onchocerca_volvulus"/>
+      <genome name="trichuris_muris"/>
+      <genome name="strongyloides_ratti"/>
+      <genome name="haemonchus_contortus_gca000469685v2wb"/>
+      <!-- Arthropoda -->
+      <genome name="nasonia_vitripennis"/>
+      <genome name="rhipicephalus_sanguineus_gca013339695v1"/>
+      <genome name="daphnia_magna_gca020631705v2"/>
+      <genome name="diabrotica_virgifera_gca003013835v2"/>
     </collection>
 
+    <collection name="metazoa">
+      <!-- outgroup -->
+      <base_collection name="shared_metazoa"/>
+      <!-- Porifera -->
+      <genome name="amphimedon_queenslandica"/>
+      <!-- Xenacoelomorpha -->
+      <genome name="hofstenia_miamia"/>
+      <!-- Placozoa -->
+      <genome name="trichoplax_adhaerens"/>
+      <!-- Cnidaria -->
+      <genome name="acropora_millepora_gca013753865v1"/>
+      <!-- Echinodermata -->
+      <genome name="asterias_rubens_gca902459465v3"/>
+      <genome name="lytechinus_variegatus_gca018143015v1"/>
+      <!-- Hemichordata -->
+      <genome name="saccoglossus_kowalevskii_gca000003605v1"/>
+      <!-- Chordata -->
+      <genome name="branchiostoma_lanceolatum"/>
+      <genome name="ciona_intestinalis"/>
+      <!-- Vertebrata -->
+      <genome name="xenopus_tropicalis"/>
+      <genome name="oryzias_latipes"/>
+      <genome name="sus_scrofa"/>
```

Comment:
I would add the mouse rather than the pig here. Again this will add very little diversity when present with the human. However, the mouse is one of the two best-annotated genomes (structural and functional) in eukaryotes so it would make sense to have it here.

Reply:
The reason for choosing Pig was its annotation recently had a big update and likely no further updates in the coming year at least. Avoiding mouse due to the impression of being updated more regularly and therefore would cause increased recomputes ?